app/assets/stylesheets/mobile.css.scss

```diff
@@ -289,3 +294,14 @@ div.note_wrapper p {
     text-align: right;
 }
 
+.btn {
+  padding: 8px 8px;
```

Comment:
Properties should be ordered background-color, background-image, border, border-radius, color, display, padding, white-space<br>Shorthand form for property `padding` should be written more concisely as `8px` instead of `8px 8px`